### PR TITLE
Ceph page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

**Blocked until we have the link to the resource mentioned [here](https://docs.google.com/a/canonical.com/document/d/17VnBygL00hXjLSECXMo3bF7KOCT5oZGFBLo6Qd6rF3A/edit?disco=AAAADuiU660)**
- Added /ceph page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/ceph
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the the page matches the [design](https://github.com/canonical-web-and-design/web-squad/issues/1945#issuecomment-548870149) and the [copy doc](https://docs.google.com/document/d/17VnBygL00hXjLSECXMo3bF7KOCT5oZGFBLo6Qd6rF3A/edit?ts=5dbb1139#heading=h.52p52stxt66j)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1946

### CODE REVIEW

**Comment by anasereijo** [comment]
@sowasred2012 looking good! just some tiny things:

Screenshot 2019-11-04 at 10 40 40
Screenshot 2019-11-04 at 10 48 12

**Comment by anasereijo** [comment]
@SirSamTumless could you check if this can go live today without the pdf? If so, we need to hide the link from the page

**Comment by sowasred2012** [comment]
@anasereijo just updated with your feedback ✌️

**Comment by anasereijo** [comment]
+1 awesome! let's get the code reviewed

File: templates/ceph/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>What is Ceph?</h2>
+      <p>Ceph is a storage solution designed to address the object, block, and file storage needs of data centres adopting open source as the new norm for high-growth block storage, object stores and data lakes. Ceph provides enterprise scalable storage while keeping CAPEX and OPEX costs in line with the underlying bulk commodity disk prices.</p>
+    </div>
+
+    <div class="col-10 u-sv3">
```
Comment (anthonydillon) [inline templates/ceph/index.html]: Shouldn't this be in its own row?

File: templates/ceph/index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      <a href="/engage/yahoo-japan-case-study">Find out more about Yahoo! Japan’s Ceph cluster&nbsp;&rsaquo;</a>
+    </div>
+
+    <div class="col-6 p-card">
+      <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/d8841399-Wellcome_Sanger_Institute_Logo_Landscape_Digital_RGB_Full_Colour.svg" alt="Wellcome Sanger Institute logo" style="max-width: 6rem;" />
+      <hr class="u-sv1">
+      <p class="p-card__content">The Wellcome Sanger Institute initially opted for self-maintained Ceph storage., but as their capacity needs grew from 3 PB to over 20 PB, they turned to Canonical to resolve their scaling issues. With the help of Canonical support, The Institute’s latest Ceph roll out decreased from a month to just four days.</p>
+      <a href="#">Find out about Wellcome Sanger path to a cost-effective Ceph infrastructure&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/ceph/index.html]: Reminder to add link before merging

File: templates/ceph/index.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      <li class="p-list__item is-ticked">Highly reliable with no single point of failure in the cluster</li>
+      <li class="p-list__item is-ticked">Fault-tolerant through data distribution and replication </li>
+      <li class="p-list__item is-ticked">Highly secure through client authentication</li>
+      <li class="p-list__item is-ticked">Compatible with Amazon S3 and OpenStack Swift APIs</li>
+      <li class="p-list__item is-ticked">Low CAPEX by using commodity hardware</li>
+      <li class="p-list__item is-ticked">Low OPEX through failure detection, self-management and self-healing</li>
+      <li class="p-list__item is-ticked">Configurable and optimisable for cost and performance</li>
+    </ul>
+
+    <hr style="margin: 2rem 0;" />
```
Comment (anthonydillon) [inline templates/ceph/index.html]: I think there is a p-separator class used on the core page.

**Pull request merged**